main.go

```diff
@@ -216,6 +218,16 @@ func main() {
 	}
 	stepconf.Print(configs)
 
+	envRepository := env.NewRepository()
+	cmdLocator := env.NewCommandLocator()
+	cmdFactory := v2command.NewFactory(envRepository)
+	rubyCmdFactory, err := ruby.NewCommandFactory(cmdFactory, cmdLocator)
+	if err != nil {
+		failf("failed to create ruby command factory: %s", err)
+	}
+	logger := v2log.NewLogger()
+	tracker := analytics.NewDefaultTracker(v2log.NewLogger(), analytics.Properties{})
```

Comment:
```suggestion
	tracker := analytics.NewDefaultTracker(logger, analytics.Properties{})
```
There is a newly created instance in the previous line

Reply:
Indeed. Fixed.